packages/tiktok-video-element/tiktok-video-element.js

```diff
@@ -155,6 +155,14 @@ class TikTokVideoElement extends (globalThis.HTMLElement ?? class {}) {
     }
   }
 
+  connectedCallback() {
+    globalThis.addEventListener('message', this.#onMessage);
+  }
+
+  disconnectedCallback() {
+    globalThis.removeEventListener('message', this.#onMessage);
+  }
```

Comment:
```suggestion
```

this shouldn't be here I think because it's valid to remove the video from the DOM and still want to control playback. we should keep this behavior similar to the native video element.